pom.xml

```
@@ -77,7 +77,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>2.10.1</version>
+        <version>2.10.2</version>
```

```suggestion
        <version>2.10.3</version>
```

It's 2.10.3 that has gRPC 1.45.4.